src/supa/job/reserve.py

```diff
@@ -257,46 +256,46 @@ def __call__(self) -> None:
                     )
                 for target, var in (("src", Variable.SOURCE_STP), ("dst", Variable.DEST_STP)):
                     # Dynamic attribute lookups as we want to use the same code for
```

Comment:
The content of this for-loop could be moved to a separate function `select_vlan()` to make the overall method a bit more readable, and so that the vlan selection logic is in a self-contained unit. (which is then easier to test and reason about)